headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -458,14 +465,15 @@ struct monster {
     undefined field_0x189;
     undefined field_0x18a;
     undefined field_0x18b;
-    undefined field_0x18c;
-    undefined field_0x18d;
-    undefined field_0x18e;
-    undefined field_0x18f;
+    // 0x18C: Bitflags to that cause non-damaging items to trigger on the attacker
+    // after they have completed their move. (Only uses first 21 bits).
```

Comment:
This description isn't very clear. Maybe you could add an example?
Also, is "to that" a typo?